master/internal/db/postgres_experiments.go

```diff
@@ -1140,14 +1142,13 @@ FROM experiments
 WHERE id = $1`, id)
 }
 
-// ExperimentCheckpointsToGCRaw returns a JSON string describing checkpoints that should be GCed
-// according to the given GC policy parameters. If the delete parameter is true, the returned
-// checkpoints are also marked as deleted in the database.
+// ExperimentCheckpointsToGCRaw returns a comma-separated string describing checkpoints
+// that should be GCed according to the given GC policy parameters. If the delete parameter is true,
+// the returned checkpoints are also marked as deleted in the database.
 func (db *PgDB) ExperimentCheckpointsToGCRaw(
 	id int,
 	experimentBest, trialBest, trialLatest int,
-	delete bool,
-) ([]byte, error) {
+) (string, error) {
```

Comment:
bit of a nit but, may as well have this return `[]string` and let another layer do the serialization.

Reply:
then every method that calls this method would have to do the serialization? Isn't it better to do it here?